Update documents of lock view for sprint 4 (#6179)
pingcap/docs: functions-and-operators/tidb-functions.md

### MySQL compatibility

The `TIDB_VERSION` function is TiDB-specific and not compatible with MySQL. If MySQL compatibility is required, you can also use `VERSION` to get version information, but the result does not contain build details.

## TIDB_DECODE_SQL_DIGESTS

The `TIDB_DECODE_SQL_DIGESTS` function is used to query the normalized SQL statements (a form without format and parameter) corresponding to the set of SQL digests in the cluster. This function accepts 1 or 2 parameters:

> * `digests`: The string type, this parameter should conform to the format of a JSON string array, and each string in the array should be a SQL Digest.

Author: ```suggestion
* `digests`: The string type. This parameter is in the format of a JSON string array, and each string in the array is a SQL digest.
```

Reviewer: Looks kind of strange. the parameter `digest`'s type is string and `stmtTruncateLength`'s type is integer.

Author: This is what is Chinese version says: https://github.com/pingcap/docs-cn/pull/6824/files#diff-78df1cdc91d235cfa2c1cc93e95e4b307abb9ae13f9a668a053bda7560d9e551R268

Reviewer: It means *this parameter is of xxx type*, rather than *this parameter is **the** xxx type*.

> * `stmtTruncateLength`: Optional parameter, integer type, used to limit the length of each SQL statement in the returned result, and it will be truncated if it exceeds the specified length. 0 means unlimited length.

Author: ```suggestion
* `stmtTruncateLength`: An Optional parameter and the integer type. It is used to limit the length of each SQL statement in the returned result. If a SQL statement exceeds the specified length, the statement is truncated. `0` means that the length is unlimited.
```

> * `stmtTruncateLength`: Optional parameter, integer type, used to limit the length of each SQL statement in the returned result, and it will be truncated if it exceeds the specified length. 0 means unlimited length.

Reviewer: The parameter *is* of the integer type.

> * `digests`: The string type. This parameter is in the format of a JSON string array, and each string in the array is a SQL digest.
> * `stmtTruncateLength`: An Optional parameter and the integer type. It is used to limit the length of each SQL statement in the returned result. If a SQL statement exceeds the specified length, the statement is truncated. `0` means that the length is unlimited.

Reviewer: Could you fix the description about the type? For example, maybe like this:
```suggestion
* `digests`: A string. This parameter is in the format of a JSON string array, and each string in the array is a SQL digest.
* `stmtTruncateLength`: An integer (optional). It is used to limit the length of each SQL statement in the returned result. If a SQL statement exceeds the specified length, the statement is truncated. `0` means that the length is unlimited.
```

> This function returns a string, which is in the format of a JSON string array. The *i*-th item in the array is the normalized SQL statement corresponding to the *i*-th element in the `digests` parameter. If an element in the `digests` parameter is not a valid SQL digest or the system cannot find the corresponding SQL statement, the corresponding item in the returned result is `null`. If the truncation length is specified (`stmtTruncateLength > 0`), for each statement in the returned result that exceeds this length, the first `stmtTruncateLength` characters are retained and the suffix `"..."` is added at the end to indicate the truncation. If the `digests` parameter is `NULL`, the returned value of the function is `NULL`.

Author: ```suggestion
This function returns a string, which is in the format of a JSON string array. The *i*-th item in the array is the normalized SQL statement corresponding to the *i*-th element in the `digests` parameter. If an element in the `digests` parameter is not a valid SQL digest or the system cannot find the corresponding SQL statement, the corresponding item in the returned result is `NULL`. If the truncation length is specified (`stmtTruncateLength > 0`), for each statement in the returned result that exceeds this length, the first `stmtTruncateLength` characters are retained and the suffix `"..."` is added at the end to indicate the truncation. If the `digests` parameter is `NULL`, the returned value of the function is `NULL`.
```

Reviewer: It's inner the json rather than SQL. The lower `null` is better here, which is different from the `NULL` in the  last sentence.

> This function returns a string, which is in the format of a JSON string array. The *i*-th item in the array is the normalized SQL statement corresponding to the *i*-th element in the `digests` parameter. If an element in the `digests` parameter is not a valid SQL digest or the system cannot find the corresponding SQL statement, the corresponding item in the returned result is `NULL`. If the truncation length is specified (`stmtTruncateLength > 0`), for each statement in the returned result that exceeds this length, the first `stmtTruncateLength` characters are retained and the suffix `"..."` is added at the end to indicate the truncation. If the `digests` parameter is `NULL`, the returned value of the function is `NULL`.

Author: ```suggestion
This function returns a string, which is in the format of a JSON string array. The *i*-th item in the array is the normalized SQL statement corresponding to the *i*-th element in the `digests` parameter. If an element in the `digests` parameter is not a valid SQL digest or the system cannot find the corresponding SQL statement, the corresponding item in the returned result is `null`. If the truncation length is specified (`stmtTruncateLength > 0`), for each statement in the returned result that exceeds this length, the first `stmtTruncateLength` characters are retained and the suffix `"..."` is added at the end to indicate the truncation. If the `digests` parameter is `NULL`, the returned value of the function is `NULL`.
```

> This function returns a string, which is in the format of a JSON string array. The *i*-th item in the array is the statement corresponding to the *i*-th element in the `digests` parameter. If an item in the `digests` parameter is not a valid SQL digest or the system cannot find the corresponding SQL statement, the corresponding item in the returned result is `null`. If the truncation length is specified (`stmtTruncateLength> 0`), for each statement in the returned result that exceeds this length, the first `stmtTruncateLength` characters are retained and the suffix `"..."` is added at the end to indicate that truncation has occurred. If the `digests` parameter is `NULL`, the returned value of the function is `NULL`.

Reviewer 2: ```suggestion
This function returns a string, which is in the format of a JSON string array. The *i*-th item in the array is the normalized SQL statement corresponding to the *i*-th element in the `digests` parameter. If an element in the `digests` parameter is not a valid SQL digest or the system cannot find the corresponding SQL statement, the corresponding item in the returned result is `null`. If the truncation length is specified (`stmtTruncateLength > 0`), for each statement in the returned result that exceeds this length, the first `stmtTruncateLength` characters are retained and the suffix `"..."` is added at the end to indicate the truncation. If the `digests` parameter is `NULL`, the returned value of the function is `NULL`.
```

> **Note:**
>
> * Only users with the [PROCESS](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_process) privilege can use this function.
> * When `TIDB_DECODE_SQL_DIGESTS` is executed, TiDB queries the statement corresponding to each SQL digest from the statement summary tables, so there is no guarantee that the corresponding statement can always be found for any SQL digest. Only the statements that have been executed in the cluster can be found, and whether these SQL statements can be queried or not is also affected by the related configuration of the statement summary tables. For the detailed description of the statement summary table, see [Statement Summary Tables](/statement-summary-tables.md).

> > * This function has a high overhead. In queries with a large number of rows (for example, querying the full table of `information_schema.cluster_tidb_trx` on a large and busy cluster), directly using this function might cause the query to run longer time. Use it with caution.

Reviewer: ```suggestion
> * This function has a high overhead. In queries with a large number of rows (for example, querying the full table of `information_schema.cluster_tidb_trx` on a large and busy cluster), directly using this function might cause the query to run for too long. Use it with caution.
```
